PortFxCop/src/Analyzer.Utilities/CommonAccessibilityUtilities.cs

```diff
@@ -2,7 +2,7 @@
 
 using Microsoft.CodeAnalysis;
 
-namespace Microsoft.AnalyzerPowerPack.Utilities
+namespace Analyzer.Utilities
```

Comment:
Is that really the root namespace for this project? I guess I didn't notice that. No "Microsoft" at the front? @michaelcfanning, @srivatsn for opinion.

In any case would not change in this PR.